app/src/main/res/values/strings.xml

```diff
@@ -91,15 +91,15 @@
     <string name="fuck_off_button_content_description">This is the famous fuck off button.</string>
     <string name="fuck_off_dialog_title">The famous fuck off button</string>
     <string name="fuck_off_dialog_text">
-        Warning! This action cannot be undone! You will never ever hear or see this person again.\n
+        Warning! This action cannot be undone! You will NEVER hear from this person again. \n
         \n
-        Think long and hard before proceeding. You\'ve told us your preferences. We\'ve given you
-        the best match that\'s interested in you. If you are going to reject this person, we are
-        going to give you the next person down your list. If you ranked them right, you are going to
+        Please, think long and hard before proceeding. Based on your preferences, we\'ve given you
+        the best match that\'s interested in you. If you reject this person, you'll get
+        the next person on your list. If you ranked them right, you are going to
```

Comment:
"If you ranked them right, you are going to like them less." That's a great sentence to capture what the users should "get into their head."